src/components/Login/LoginForm.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Form } from "react-final-form";
+import { useTranslation } from "react-i18next";
+
+import {
+  emailRegex,
+  required,
+  composeValidators,
+} from "../../utils/validators";
+
+import EmailField from "../UI/EmailField";
+import PasswordField from "../UI/PasswordField";
+import Action from "../UI/Action";
+
+const LoginForm = (props) => {
+  const { t } = useTranslation();
+  return (
+    <Form
+      onSubmit={props.handleSubmit}
+      render={({ submitError, modifiedSinceLastSubmit, handleSubmit }) => (
+        <div className="flex justify-center mr-8">
+          <form onSubmit={handleSubmit} className="w-full max-w-sm">
+            <EmailField
```

Comment:
We can do even better:
1) translation namespace can be specified only once. `EmailField` component can do `${namespace}.${name}Placeholder` and `${namespace}.${name}` itself.
2) Email validation always applied to the EmailField so component can do that inside

So real props list is:
- namespace
- name
- required (boolean)